src/Components/DrillListPage.js

```diff
@@ -64,39 +66,25 @@ const styles = StyleSheet.create({
     height: '100%',
   },
   counter: {
-    color: theme.COLOR_SECONDARY,
-    marginBottom: 20,
+    ...list.counter,
```

Comment:
I think you don't even need to declare a counter style here.
You could use directly `list.counter` on your element since we don't change anything to it
That will reduce the boilerplate code a lot

(same for the others)